index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>HTML5 Hello World!</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+<p>Hello World!</p>
```

Comment:
#### Nitpick :warning:
You don’t have to forget about the use of indentations because it makes your code more understandable.